src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -961,6 +961,8 @@ public String getValueFromTracking(NetworkTrafficInterceptor networkTrafficInter
 
   public String getFVLineItem() {
     final By FV_SLOT_SELECTOR = By.cssSelector("div.featured-video__player-container");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)